### PR TITLE
Update arabic

### PR DESCRIPTION
**Pull request by UltraHDR**

### CODE REVIEW

File: streams/iq.m3u [hunk 1]
```diff
@@ -85,3 +85,7 @@ https://5a3ed7a72ed4b.streamlock.net/zagrostv/SMIL:myStream.smil/playlist.m3u8
 https://live.karwan.tv/karwan.tv/waar-tv/playlist.m3u8
 #EXTINF:-1 tvg-id="" status="timeout",Music Al-Remas (720p) [Not 24/7]
 http://free.fullspeed.tv/iptv-query?streaming-ip=https://www.youtube.com/c/musicalrmastv1/live
+#EXTINF:-1 tvg-id="AlIraqiyaAl3ama.iq" status="online",Al Iraqiya Al 3ama (720p) [Not 24/7]
+https://cdn.catiacast.video/abr/8d2ffb0aba244e8d9101a9488a7daa05/playlist.m3u8
```
Comment (LaneSh4d0w) [inline streams/iq.m3u:89]: 401 Authorization Required

File: streams/iq.m3u [hunk 2]
```diff
@@ -85,3 +85,7 @@ https://5a3ed7a72ed4b.streamlock.net/zagrostv/SMIL:myStream.smil/playlist.m3u8
...
+#EXTINF:-1 tvg-id="AlIraqiyaNews.iq" status="online",Al Iraqiya News (720p) [Not 24/7]
+https://cdn.catiacast.video/abr/78054972db7708422595bc96c6e024ac/playlist.m3u8
```
Comment (LaneSh4d0w) [inline streams/iq.m3u:91]: Same on this one

File: streams/iq.m3u (hunk 1 above)
Comment (UltraHDR) [reply streams/iq.m3u:89]: Added [Geo-blocked] to the title

File: streams/iq.m3u (hunk 2 above)
Comment (UltraHDR) [reply streams/iq.m3u:91]: Added [Geo-blocked] to the title

File: streams/iq.m3u [hunk 3]
```diff
@@ -85,3 +85,7 @@ https://5a3ed7a72ed4b.streamlock.net/zagrostv/SMIL:myStream.smil/playlist.m3u8
 https://live.karwan.tv/karwan.tv/waar-tv/playlist.m3u8
```
Comment (LaneSh4d0w) [inline streams/iq.m3u:85]:
```suggestion
#EXTVLCOPT:http-referrer=https://ku.karwan.tv/
https://live.karwan.tv/karwan.tv/waar-tv/playlist.m3u8
```

**Review by freearhey — DISMISSED** [review]

**Review by freearhey — APPROVED** [review]

**Review by BellezaEmporium — APPROVED** [review]

**Pull request merged**